src/fragments/lib/datastore/ios/sync/40-clear.mdx

```diff
@@ -1,6 +1,6 @@
 <BlockSwitcher>
 
-<Block name="Listener (iOS 11+)">
+<Block name="Async/Await">
 
 ```swift
 let isSignedOut = HubFilters.forEventName(HubPayload.EventName.Auth.signedOut)
```

Comment:
Does the `clear` need to be `async` as well in the snippet? Same comment in combine block

Reply:
yes, it needs to be, I have doubts in this file so left it as is -> 
here the Xcode is prompting for `listner: ` in `Amplify.Hub.listen(to: .auth, isIncluded: isSignedOut)`. I'm not sure how to define a `HubListner`